keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1104,13 +1305,14 @@ Pods that has already been created will stay being scheduled after the downgrade
 not be rejected by control
 plane nor by kubelet. Both will treat the sidecar container as an Init container.
 This may render the Pod unusable as it will stuck in initialization forever -
-sidecar container are never exiting. We will document this behavior for Alpha release.
-Promoting feature to Beta we will revisit the situation. If we will see this as
-a major issue, we will need to wait for 3 releases so kubelet will have the logic
+sidecar container are never exiting.
+This behavior has been documented for Alpha release, but we don't see it as a
+major issue requiring to wait for 3 releases so kubelet will have the logic
 to reject such Pods when the feature gate is disabled to keep Downgrade safe.
 
-**Note**, For the control plane and kubelet we will implement logic to reject Pods
-with sidecar containers when feature gate got turned off.
+**Note**, We have implemented logic for the kubelet to reject Pods
+with sidecar containers when feature gate is turned off. For the control plane
+we simply ignore the new field to maintain Pod scheduling.
```

Comment:
Let's be more explicit what exactly is happening as mentioned by @gjkim42

For the control plane - kube-apiserver is dropping the field (if it wasn't set before) and kube-scheduler is keeping pods with the field set unschedulable.

Reply:
nit: linking the code you pasted in this comment thread:
https://github.com/kubernetes/enhancements/pull/4255#discussion_r1346321388

would be extremely helpful here too